tests/saitests/sai_qos_tests.py

```diff
@@ -736,7 +736,7 @@ def runTest(self):
                                     ip_dst=dst_port_ip,
                                     ip_tos=tos,
                                     ip_ttl=ttl)
-            send_packet(self, src_port_id, pkt, pkts_num_leak_out + pkts_num_trig_pfc - pkts_num_dismiss_pfc - hysteresis)
+            send_packet(self, src_port_id, pkt, pkts_num_leak_out + pkts_num_trig_pfc - pkts_num_dismiss_pfc - hysteresis - margin)
```

Comment:
This change can affect other platforms

Reply:
Indeed it can affect other platform slightly. By default, the margin is 1 (for Mellanox platform it can be up to 3) so I suppose the affection is acceptable.
How do you think? @neethajohn
Or we need to introduce another variable to represent it

Reply:
I think we need to handle it differently for Mellanox because in this particular test with the margin, we might end up triggering PFC when we sent packets to the second port itself

Reply:
ok. understood. will handle it within Mellanox code.